libs/infinity_emb/pyproject.toml

```diff
@@ -28,6 +28,7 @@ transformers = {version = ">4.8.0", optional=true}
 ctranslate2 = {version = "^4.0.0", optional=true}
 optimum = {version = ">=1.16.2", optional=true, extras=["onnxruntime"]}
 hf_transfer = {version=">=0.1.5"}
+einops = {version = "^0.8.0", optional=true}
```

Comment:
Should this be a extra group of its own or should it just be part of something like `torch` ?

Reply:
Extra group is good. I would say it might be better to pin with “*” as version and let it auto resolve.